microsite/data/plugins/aws-app-development.yaml

```diff
@@ -1,10 +1,10 @@
 ---
-title: AWS App Developer Tools
+title: OPA on AWS
 author: Amazon Web Services
 authorUrl: https://aws.amazon.com/
 category: Infrastructure
-description: Create and manage AWS Apps within Backstage
-documentation: https://github.com/awslabs/app-development-for-backstage-io-on-aws#readme
-iconUrl: https://github.com/awslabs/app-development-for-backstage-io-on-aws/blob/main/docs/images/AWS_logo.png?raw=true
+description: Orchestrate Platforms and Applications (OPA) allows customers to build and manage AWS Apps & Environments within Backstage
+documentation: https://opaonaws.io
```

Comment:
This just send me to a landing page about OPA, if I want to learn how to actually install the Backstage plugin it's not clear how to do that. Is there perhaps a better link we could use? Or maybe keep the current one?

Reply:
The OPA on AWS suite of backstage plugins needs an AWS backend that is provisioned as well - it wouldn't be sufficient to install backstage plugins only for this solution. 
We have full documentation on this link - https://opaonaws.io/docs/getting-started/deploy-the-platform
should we update it to this one?

Reply:
Guess I'm just find it really hard to understand how to install your plugin from that site, even with the link you just shared. Are you saying the only way to install is to clone that repo? That's for sure not a common way to install plugins that most adopters will expect. What if I already have a Backstage instance?

Reply:
Our plugins are published to npm so the installation process is standard. However, we introduce a new kind and built UI pages and cards, as a result we needed to reroute requests from EntityPage.tsx to pages exists in our plugins.
  ```
 <EntitySwitch>
     <EntitySwitch.Case if={isKind('component')} children={componentPage} />
+    {/* AWS Modifications */ }
+    <EntitySwitch.Case if={isKind('resource')} children={resourceEntityPage} />
+    <EntitySwitch.Case if={isKind('awsenvironment')} children={awsEnvironmentEntityPage} />
+    <EntitySwitch.Case if={isKind('awsenvironmentprovider')} children={awsEnvironmentProviderEntityPage} />
+    {/* End AWS Modifications */ }

```
 in addition we have a integrated a few other UI elements(AWS catalog ribbon) that requires to update the backstage Sidebar
The process is fully documented here - 
https://github.com/awslabs/app-development-for-backstage-io-on-aws/blob/main/docs/PLUGIN_INSTALL.md

The deployment process described in our website fully automate this using a patch 
https://github.com/awslabs/app-development-for-backstage-io-on-aws/tree/main/backstage-mods

It is possible to use the plugins on an existing backstage but users will need to apply the patch on the existing backstage image.

Appreciate your help and support.

Reply:
Right, but all of this isn't super easy for some one to know, at least I had a hard time and only understand better because you helped me. We face the same challenges with Backstage itself.

When I look at PRs for adding/updating plugins in the Plugin Directory I try my best to pretend to be someone who might use a given plugin. Where i can i try to offer some advice where needed so you are more likely to see your plugin(s) getting used. I'm not trying to prevent these changes going forward though so I'm going to approve this just keep in ind that it would be great to make the docs easier to consume for people brand new to your plugin 👍

Reply:
Agree. and thank you for the feedback - we should (and can) do a better job in explaining how these plugins works. We have in our backlog to decouple some of the work into several plugins so that the process will be easier to install and use.

Appreciate your support and approving this MR 🙏